requirements.txt

```diff
@@ -0,0 +1,4 @@
+Sphinx<5.0.0
+furo<2022.4.8
+myst-parser<0.18
+sphinx-lint<1
```

Comment:
For these dependencies, let's do an exact pin (e.g. `sphinx-lint==0.6.0`). This will ensure that updates to these dependencies do not break our builds from underneath of us. Furthermore, we do not really need to use ranges as generally these builds will be in their own separate virtualenv and not necessarily need to be used with another project. So we can be strict on the version.